### PR TITLE
fix: track lines in journey and removing unused stories

### PR DESCRIPTION
**Pull request by christian14b**
Screenshot 2024-12-12 at 11 06 33 AM

### CODE REVIEW

File: .storybook/stories/BadgeContainer.stories.js [hunk 1]
```diff
@@ -67,8 +67,5 @@ export const RectangleInProgress = story({ status: BADGE_IN_PROGRESS, shape: BAD
 
 export const RectangleLocked = story({ status: BADGE_LOCKED, shape: BADGE_SHAPE_RECTANGLE }, badgeUsEconomicEquality.contentfulData[0].imageUrl);
 
-export const BadgeWithRays = story({showRays: true, status: BADGE_COMPLETED, shape: BADGE_SHAPE_ARCH}, badgeClimateAction.contentfulData[0].imageUrl);
-
 export const BadgeWithShine = story({showShine: true, status: BADGE_COMPLETED, shape: BADGE_SHAPE_ARCH}, badgeClimateAction.contentfulData[0].imageUrl);
 
-export const BadgeWithShineAndRays = story({showRays: true, showShine: true, status: BADGE_COMPLETED, shape: BADGE_SHAPE_ARCH}, badgeClimateAction.contentfulData[0].imageUrl);
```
Comment (mcstover) [inline .storybook/stories/BadgeContainer.stories.js:74]: Are these no longer applicable?

File: src/components/MyKiva/BadgeContainer.vue [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <template>
 	<div
-		class="tw-relative tw-inline-block tw-w-auto tw-overflow-hidden"
```
Comment (roger-in-kiva) [inline src/components/MyKiva/BadgeContainer.vue:3]: `tw-overflow-hidden` is already used in TY page, can we add a prop for that case?

Comment (roger-in-kiva) [reply src/components/MyKiva/BadgeContainer.vue:3]: Well, it's used for shining effect, probably should be added under that condition...

Comment (roger-in-kiva) [reply src/components/MyKiva/BadgeContainer.vue:3]: Looks like fixed for all cases we have.

Is this case no longer available?
Screenshot 2024-12-12 at 2 49 54 p m

**Review by mcstover — APPROVED** [review]

**Review by roger-in-kiva — APPROVED** [review]

**Pull request merged**